pyro/compressible/interface.py

```diff
@@ -201,8 +201,8 @@ def states(idir, ng, dx, dt,
 
             # construct the states
             for m in range(nvar):
-                sum_l = np.dot(betal, rvec[:, m])
-                sum_r = np.dot(betar, rvec[:, m])
+                sum_l = np.dot(betal, np.ascontiguousarray(rvec[:, m]))
```

Comment:
was this causing issues? or did you just want to make it explicit?

Reply:
numba was just complaining that it would be faster if we work with contiguousarray, so I just changed it.